source/default_cmm.c

```diff
@@ -180,3 +185,10 @@ int aws_cryptosdk_default_cmm_set_alg_id(struct aws_cryptosdk_cmm *cmm, enum aws
     self->default_alg = alg_id;
     return AWS_OP_SUCCESS;
 }
+
+bool aws_cryptosdk_default_cmm_is_valid(const struct aws_cryptosdk_cmm *cmm) {
+    AWS_PRECONDITION(cmm != NULL);
```

Comment:
It should return false in case `cmm` is NULL, not fail on an assertion.

Reply:
Right, I am happy you saw this because I had my doubts about it.